src/constants.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  API_VERSION: 2,
+  MAX_STRING_LENGTH: 1024,
```

Comment:
Let's increase this to 4096. In the future we may make the backend support larger strings, so we don't want the SDK to truncate them early.